### PR TITLE
Simple search screen updated

### PR DESCRIPTION
**Pull request by Lumiere-Lodi**
This PR has updated the simple search screen with the new Design.

### CODE REVIEW

File: src/components/search/SimpleSearchRow.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  index,
+  onChange
+}) => {
+  
+  const options = [
+    'Exact',
+    'Low Fuzziness',
+    'Medium Fuzziness',
+    'High Fuzziness'
+  ]
```
Comment (marrouchi) [inline src/components/search/SimpleSearchRow.tsx:28]: Let's have the options rather an array of objects :
```suggestion
  const options = [
    { value: 0, label: 'Exact' },
    { value: 1, label: 'Low Fuzziness'},
    { value: 2, label: 'Medium Fuzziness'},
    { value: 3, label: 'High Fuzziness'}
  }]
```

**Review by marrouchi — CHANGES_REQUESTED** [review]
Looks good, left a comment.

File: src/components/search/SimpleSearchRow.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+  const [matchType, setMatchType] = useState<string>('')
+
+  function handleStrictLevelChange(event: React.ChangeEvent<any>) {
```
Comment (IlyesBenAmara) [inline src/components/search/SimpleSearchRow.tsx:32]: Let us use the arrow function syntax :wink:

**Review by IlyesBenAmara — CHANGES_REQUESTED** [review]
LGTM, left you some comments :technologist: :wink:

File: src/components/search/SimpleSearchRow.tsx (hunk 2 above)
Comment (Lumiere-Lodi) [reply src/components/search/SimpleSearchRow.tsx:32]: Thanks. 
Just out of curiosity, is there a specific reason for that?

File: src/components/search/SimpleSearchRow.tsx (hunk 1 above)
Comment (Lumiere-Lodi) [reply src/components/search/SimpleSearchRow.tsx:28]: yay. Thanks

File: src/components/search/SearchSelectField.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+const SearchSelectField: React.FC<SearchSelectFieldProps> = ({
+  onChange,
+  fieldName,
+  options,
+  title,
+  description,
+  helperText,
+  sx = { width: 200 },
```
[the remark this answers is not in the document]
Comment (Lumiere-Lodi) [reply src/components/search/SearchSelectField.tsx:30]: I 100% agree with you. I tried that with maxWidth.  It did not work out well. 

I had the same concern until we established we shouldn't worry about responsiveness at this point. Thats why it is kept this way. 

Thanks.

File: src/components/search/SimpleSearchRow.tsx [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        )}
+      </Grid>
+      <Grid>
+        <SearchDropdown
+          options={options}
+          onChange={handleStrictLevelChange}
+          index={index}
+          title={'Match Type'}
+          description={'Select Match Type'}
+          sx={{ width: 220, ml: 2, mt: 0.5 }}
```
[the remark this answers is not in the document]
Comment (Lumiere-Lodi) [reply src/components/search/SimpleSearchRow.tsx:88]: replied above. Thank you.

File: src/components/search/SimpleSearchRow.tsx (hunk 2 above)
Comment (IlyesBenAmara) [reply src/components/search/SimpleSearchRow.tsx]: It is just used for consistency, that's all :smile_cat:

**Review by IlyesBenAmara — APPROVED** [review]
Nice work @Lumiere-Lodi

**Review by marrouchi — APPROVED** [review]
Awesome !

**Pull request merged**